docfx.json

```diff
@@ -508,6 +519,7 @@
             "recommendations": {
                 "_csharplang/**/*.md": false,
                 "_csharpstandard/**/*.md": false,
+                "_roslyn/docs/compilers/CSharp/*.md": "true",
```

Comment:
`true` is the default global value:

https://github.com/dotnet/docs/blob/4230e45924d2457846a0196ddd0bf65b5c9c072b/docfx.json#L117

I think only documents where we want to disable recommendations are added here

Reply:
Good point:

```suggestion
```